pkg/varlinkapi/container.go

```diff
@@ -847,7 +847,7 @@ func GenerateRunlabelCommand(runLabel, imageName, name string, opts map[string]s
 func envSliceToMap(env []string) map[string]string {
 	m := make(map[string]string)
 	for _, i := range env {
-		split := strings.Split(i, "=")
+		split := strings.SplitN(i, "=", 2)
 		m[split[0]] = strings.Join(split[1:], " ")
```

Comment:
This one if funky, but actually OK.  I just double checked in the golang sandbox, it returns empty due to the colon `:`.  Not my knowledge, I originally had a check in here, and @nalind recommended this to make it sleeker.

Reply:
Since there is only two sections, this should be 
```
if len(split) > 1 {
     m[split[0]=split[1]
}
```

Reply:
This one doesn't need to change.  Due to the split[1:], it handles things just fine.  I believe this change might be the cause of my last test failure.  reverting.

Reply:
Hm, but if you leave it as it is, won't it replace all `=` with spaces? I'm not familiar enough with the surrounding code to know if that's intentional.

Reply:
@TomSweeneyRedHat I understand it will work, but the call is very confusing to someone coming upon this for the first time.

@xordspar0 No the only thing that will happen is the strings.Join(split[:1], " ") will return split[1] if it exists or return "" if it does not.  The join would have added the spaces if the list of split[:1] is longer then one, but this can never happen after the change, thus the code is hard to understand.

Reply:
I agree that it works after the change. However, I believe it was reverted back to the original in the latest commit to this branch:

https://github.com/containers/podman/blob/33366a2cda518fafed0e72cf6e53b55abfa3c5e5/pkg/varlinkapi/container.go#L849-L852